assets/sitedata/menus.json

```
@@ -48,8 +48,8 @@
 					"separator": "true"
 				},
 				{
-					"title": "Google Summer of Code 2024",
-					"url": "https://owasp.org/gsoc/gsoc2024"
+					"title": "Google Summer of Code 2025",
+					"url": "https://owasp.org/gsoc/gsoc2025"

```

Let's make it generic so that we don't have this issue next year.
```suggestion
					"title": "Google Summer of Code",
					"url": "https://owasp.org/gsoc"

```

---

It's better long term, I'm okay with losing the year details.